### PR TITLE
[FBZ-10398] - Migrate elasticsearch cookbook to stack v7

### PR DESCRIPTION
**Pull request by rupikakapoor**
**Description of your patch**
Install and configure Elasticsearch by migrating recipe from v6 to v7

**Recommended Release Notes**
Added support for Elasticsearch installation 

**Estimated risk**
Medium

**Components involved**
ey-elasticsearch

**Dependencies**
none

**Description of testing done**

* Create a new environment with v7 as stack
* Uploaded custom-elasticsearch recipe as per process defined on https://github.com/engineyard/ey-cookbooks-stable-v7/tree/fbz10398_migrate_elasticsearch_recipe/custom-cookbooks/elasticsearch/cookbooks/custom-elasticsearch#elasticsearch 
* Added utility instance named `elasticsearch`.
* SSH into instance check for elasticsearch service running using `systemctl status elasticsearch.service` version installed  and verify version using `curl localhost:9200`. 

**QA Instructions**
* Create a new environment with v7 as stack
* upload `custom-elasticsearch` recipe as per process defined on https://github.com/engineyard/ey-cookbooks-stable-v7/tree/fbz10398_migrate_elasticsearch_recipe/custom-cookbooks/elasticsearch/cookbooks/custom-elasticsearch#elasticsearch 
* Add utility instance named `elasticsearch` and allow it to boot.
* SSH into instance check for elasticsearch service running using `systemctl status elasticsearch.service` version installed  and verify version using `curl localhost:9200`.

### CODE REVIEW

File: custom-cookbooks/elasticsearch/cookbooks/custom-elasticsearch/attributes/default.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  )
+
+  # Run Elasticsearch on a solo or app_master instance
+  # Not recommended for production environments
+  #elasticsearch['is_elasticsearch_instance'] = ( ['solo', 'app_master'].include?(node['dna']['instance_role']) )
+
+  # Elasticsearch version to install
+  # Go to https://www.elastic.co/downloads/past-releases to see the available version
+  # elasticsearch['version'] = '5.6.15'
+  elasticsearch['version'] = 6.6.3'
```
Comment (Kyle-Law) [inline custom-cookbooks/elasticsearch/cookbooks/custom-elasticsearch/attributes/default.rb]: @rupikakapoor  should it be 

```rb
elasticsearch['version'] = '6.6.3'
```

?

lacking a `'` before 6.6.3

**Review by mushyy — CHANGES_REQUESTED** [review]
Some errors due to missing closing `'` and needs reviewing through linting process

File: custom-cookbooks/elasticsearch/cookbooks/custom-elasticsearch/attributes/default.rb (hunk 1 above)
Comment (rupikakapoor) [reply custom-cookbooks/elasticsearch/cookbooks/custom-elasticsearch/attributes/default.rb]: Thank you, this is fixed now

**Comment by kvbautista** [comment]
This recipe is now being tested on an environment.

Please update version `elasticsearch['version'] = 6.6.3'` to one of the listed below:

```
elasticsearch |      6.8.0
elasticsearch |      6.7.2
elasticsearch |      6.7.1
elasticsearch |      6.7.0
elasticsearch |      6.6.2
elasticsearch |      6.6.1
elasticsearch |      6.6.0
```

As 6.6.3 is not an available version for the package in ubuntu20.04

**Comment by mushyy** [comment]
LGTM will merge for QA

**Pull request merged**